corehq/apps/api/resources/v0_5.py

```diff
@@ -1415,9 +1415,7 @@ def obj_get(self, bundle, **kwargs):
         domain = kwargs['domain']
         if not toggles.SUPERSET_ANALYTICS.enabled(domain):
             raise ImmediateHttpResponse(
-                HttpForbidden(gettext_noop(
-                    "This domain requires the superset-analytics flag to access this endpoint."
-                ))
+                HttpForbidden()
```

Comment:
Thanks for the follow up.

Apologies for the confusion here though. We should be returning a 404 and not 403, as shared in the link reference I shared about what HQ does by default when feature flag isn't enabled.

Reply:
See https://github.com/dimagi/commcare-hq/pull/34987/commits/0bc6162f89e6a160534d0f1693736cec7d0b0df3